app/models/generic_event/move_reject.rb

```diff
@@ -1,42 +1,22 @@
 class GenericEvent
   class MoveReject < GenericEvent
-    DETAILS_ATTRIBUTES = %w[
-      rejection_reason
-      cancellation_reason_comment
-      rebook
-    ].freeze
+    details_attributes :rejection_reason, :cancellation_reason_comment, :rebook
 
     include MoveEventValidations
 
     validates :rejection_reason, inclusion: { in: Move::REJECTION_REASONS }
 
-    def rejection_reason
-      details['rejection_reason']
-    end
-
-    def rejection_reason=(reason)
-      details['rejection_reason'] = reason
-    end
-
-    def cancellation_reason_comment
-      details['cancellation_reason_comment']
-    end
-
-    def rebook?
-      details.fetch('rebook', false)
-    end
-
     def trigger
       eventable.status = Move::MOVE_STATUS_CANCELLED
       eventable.rejection_reason = rejection_reason
       eventable.cancellation_reason = Move::CANCELLATION_REASON_REJECTED
       eventable.cancellation_reason_comment = cancellation_reason_comment
-      eventable.rebook if rebook?
+      eventable.rebook if rebook
     end
 
     def for_feed
       super.tap do |common_feed_attributes|
-        common_feed_attributes['details']['rebook'] = rebook?
+        common_feed_attributes['details']['rebook'] = rebook
```

Comment:
Did you want to add `|| false` here so it's always a boolean value in the feed rather than potentially `nil` ?

Reply:
Yeah, we should surface it.

Reply:
Surfaced in most recent commit :)